Flexible Learning/units_browse_details.php

```diff
@@ -192,13 +192,25 @@
                 ]]
             ]));
 
-            echo $form->getOutput(); 
+            echo $form->getOutput();
         }
     }
 
     // SMART BLOCKS
     $blocks = $unitBlockGateway->selectBlocksByUnit($flexibleLearningUnitID)->fetchAll();
 
+    if ($values['outline'] != NULL) {
+      $outlineBlock = [
+        'flexibleLearningUnitBlockID' => 0,
+        'flexibleLearningUnitID' => $flexibleLearningUnitID,
+        'title' => 'Unit Outline',
+        'type' => NULL,
+        'length' => NULL,
+        'contents' => $values['outline'],
+        'teachersNotes' => NULL,
+        'sequenceNumber' => NULL];
+      array_unshift($blocks, $outlineBlock);
+    };
```

Comment:
A quick note, for consistency, that null does not need to be capitalized.

```suggestion
    if ($values['outline'] != null) {
      $outlineBlock = [
        'flexibleLearningUnitBlockID' => 0,
        'flexibleLearningUnitID' => $flexibleLearningUnitID,
        'title' => 'Unit Outline',
        'type' => null,
        'length' => null,
        'contents' => $values['outline'],
        'teachersNotes' => null,
        'sequenceNumber' => null];
      array_unshift($blocks, $outlineBlock);
    };
```